.travis.yml

```diff
@@ -6,10 +6,13 @@ scala:
 jdk:
   - oraclejdk8
 env:
-  # Setting this variable twice will cause the 'script' section to run twice with the respective env var invoked
-  - BUILD_TYPE=sbt
-  - BUILD_TYPE=centaurJes
-  - BUILD_TYPE=centaurLocal
+  global:
+    - CENTAUR_BRANCH=79dc5f9f0f6ca6aedb5b13358d47b0c9c4156d51
```

Comment:
Would it be more or less confusing to call it `CENTAUR_TREEISH` as that's really what it is?